suprb/suprb.py

```diff
@@ -14,7 +14,7 @@
 from .optimizer.rule import RuleGeneration
 from .optimizer.rule.es import ES1xLambda
 from .rule import Rule
-from .rule.matching import MatchingFunction, OrderedBound
```

Comment:
Do we need those imports here? I think we only need the default one here and the others are specified in the runs itself